src/vs/platform/terminal/common/capabilities/commandDetectionCapability.ts

```diff
@@ -363,6 +364,11 @@ export class CommandDetectionCapability extends Disposable implements ICommandDe
 		this._logService.debug('CommandDetectionCapability#handleCommandStart', this._currentCommand.commandStartX, this._currentCommand.commandStartMarker?.line);
 	}
 
+	override dispose() {
+		super.dispose();
+		this._isDisposed = true;
+	}
```

Comment:
We want to avoid overriding dispose, you can access `_isDIsposed` off `Disposable`

Reply:
I tried that (`this._isDisposed and super._isDisposed`) and neither were accessible.

Reply:
oh I mean `this._store.isDisposed`